lib/libyui_client/widgets/numberbox.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module LibyuiClient
+  module Widgets
+    # Class representing a numberbox in the UI. It can be YIntField.
+    class Numberbox < Widgets::Base
+      # Returns minimum value to set in the numberbox
+      # @return [Integer] with minimum value
+      # @example Get minimum value to set numberbox with id 'test'
+      #   {
+      #      "class": "YIntField",
+      #      "debug_label": "label_test",
+      #      "hstretch": true,
+      #      "id": "test",
+      #      "label": "label_test",
+      #      "max_value": 65535,
+      #      "min_value": 0,
+      #      "value": 3260
+      #   }
+      #   app.numberbox(id: 'test').min_value
```

Comment:
Could you please add `@example` tag before the `app.numberbox(id: 'test').min_value`, so that it will be shown as separate block in documentation. There are several places where it is missed. I'll not list all of them to not pollute the thread with same comments, could you please fix them all.

Probably it is known to you, but just a friendly reminder. You can run `yard` in the root of the project and then open `index.html` file in browser to observe if your doc is properly rendered.

Reply:
I'm going to try to run it in my setup.

Reply:
I discovered a lot of silly mistakes that I did related with visualization, good advice, docs with yard looks better now.